admin/config.yml

```diff
@@ -49,8 +52,59 @@ collections:
         file: "_data/team-alpha.json"
         description: "Team Alpha Members List"
         fields:
-          - {label: "Name", name: "name", widget: "string"}
-          - {label: "Rule", name: "rule", widget: "string"}
-          - {label: "Song", name: "song", widget: "string"}
-          - {label: "Person", name: "person", widget: "string"}
-          - {label: "Fact", name: "fact", widget: "string"}
+          - label: Entries
```

Comment:
@mhisle22 Can we change the label to something more descriptive? Like "Team Alpha Members"